tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -0,0 +1,477 @@
+@pytest.mark.kvmagent
+class TestMevocoResetDefaultGateway:
+    def test_reset_default_gateway_success(self):
+        plugin = _make_plugin()
+        tmp_file = tempfile.NamedTemporaryFile(delete=False)
+        tmp_file.close()
+
+        setattr(
+            plugin,
+            "_make_conf_path",
+            MagicMock(return_value=("/tmp/conf", "/tmp/dhcp", "/tmp/dns", tmp_file.name, "/tmp/log")),
+        )
```

Ruff (0.15.2)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)